### DISCUSSION TITLE
dateTimeFormat for Calendar using localization

### DISCUSSION THREAD

**Discussion by rui-filipe-ribeiro** [primefaces/community#808] [PrimeVue] [answered]
How to set date/time format using localization in Calendar component?
We have the **dateFormat** variable that only defines the date, but how to define date/time in the locale? 
Shouldn't there be a **dateTimeFormat24** and **dateTimeFormat12** set, for example?

Look this example:
`<Calendar id="calendar-24h" v-model="datetime24h" showTime hourFormat="24" />`

**Comment by tugcekucukoglu** [comment, marked as answer]
`dateFormat` can be set https://primevue.org/calendar/#api.calendar.props.dateFormat and also please check for more information https://primevue.org/calendar/#locale

**Reply by rui-filipe-ribeiro** [reply]
I appreciate your answer, but I think the problem is that the component does not format correctly when the v-model has a value in the ISO 8601 format (ex: 2024-01-04T16:50:01.427). Wasn't it supposed to use formatDate? Thanks.
